src/app/service/task.service.ts

```diff
@@ -1,23 +1,53 @@
 import { Injectable } from '@angular/core';
-import { Task } from '../task';
+import { Task, tempTask } from '../models/task.model';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+// import tasksdata from '../../assets/tasks.json';//mēģināju ielikt no assets foldera, bet nekādīgi nestrādāja//sanāca
+
 
 @Injectable({
   providedIn: 'root'
 })
 export class TaskService {
-  private tasks: Task[] = [];
+  constructor(private http: HttpClient){}
+
```

Comment:
format the code please, you can setup prettier in vscode (extension) to make it format your code after saving changes